js/src/forum/components/Composer.js

```diff
@@ -56,7 +56,8 @@ export default class Composer extends Component {
     );
   }
 
-  onupdate() {
+  onupdate(vnode) {
+    super.onupdate(vnode);
```

Comment:
Do we also want an additional empty line below the `super` call like all the other instances for consistent formatting? Same thing on line 100 of this file.